src/test-helpers/initStore.ts

```diff
@@ -0,0 +1,18 @@
+import clearStore from './clearStore'
+import skipTutorialHelper from './skipTutorial'
+
+interface Params {
+  persist?: boolean
```

Comment:
You can add JSDOCs to `persist` and `allowTutorial`. Then they will provide helpful popups when using IntelliSense.